src/test/java/PawnTest.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PawnTest {
+
+    @Test
+    public void create(){
+        Pawn whitePawn = verifyPawn("white");
+        Pawn blackPawn = verifyPawn("black");
```

Comment:
여기서 아래와 같이 바로 생성자를 호출하셔도 되겠네요.

```suggestion
        Pawn whitePawn = new Pawn("white");
        Pawn blackPawn = new Pawn("black");
````

Reply:
요구사항에서 veritfyPawn 메소드로 코드 중복을 제거하라는 것을 잘 이해하지 못하고 코드를 작성한 것 같습니다. 해당 메소드를 수정해보겠습니다!